SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/ViewModel/RateItViewModel.cs

```diff
@@ -247,6 +247,25 @@ private void EditorController_ActiveDocumentChanged(object sender, DocumentEvent
 			ActiveDocument.ActiveSegmentChanged += ActiveDocument_ActiveSegmentChanged;
 		}
 
+		/// <summary>
+		/// We are doing this when the user forces the sending of the feedback(clicks on SendFeedback) without any improvement, rating or feedback message
+		/// because a feedback cannot be sent without any info so we're adding the original target itself as a suggestion
+		/// </summary>
+		/// <param name="segmentId">When this is null the user clicked on SendFeedback instead of it being sent automatically</param>
+		/// <param name="suggestion"></param>
+		/// <param name="rating"></param>
+		/// <returns></returns>
+		private string EnsureFeedbackWillGetThrough(SegmentId? segmentId, ImprovementFeedback suggestion, dynamic rating)
```

Comment:
Is it possible to remove suggestion != null and use suggestion?.Improvement == null

Reply:
done

Reply:
done